src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,22 @@
-export const PostInfo = () => <>Put the post here</>;
+import { UserInfo } from "../UserInfo/UserInfo";
+import { CommentList } from "../CommentList/CommentList";
+
+export const PostInfo = ({post}) => <>
+  <div className="PostInfo">
+  <div className="PostInfo__header">
+    <h3 className="PostInfo__title">{post.title}</h3>
+    <UserInfo user={post.user}/>
+  </div>
+
+  <p className="PostInfo__body">
+    {post.body}
+  </p>
+  {
+    post.comments.length > 0?
```

Comment:
Consider adding a check to ensure `post.comments` is defined before accessing its `length` property. This will prevent potential runtime errors if `post.comments` is `undefined`.